### PR TITLE
Edit private routes

### PR DESCRIPTION
**Pull request by PavelEvleev**
All routes are placed in one place

### CODE REVIEW

File: web/src/components/Home/DrawerOpenRightExample.jsx [hunk 1]
```diff
@@ -59,28 +59,50 @@ class DrawerOpenRightExample extends React.Component {
   }
   
   render() {
+
+    if(this.props.login){
+      return (
+        <div>
```
Comment (lehaSVV2009) [inline web/src/components/Home/DrawerOpenRightExample.jsx]: remove unused div

File: web/src/components/Home/DrawerOpenRightExample.jsx [hunk 2]
```diff
@@ -4,7 +4,7 @@ import AppBar from 'material-ui/AppBar'
 import MenuItem from 'material-ui/MenuItem';
 import ArrowDropRight from 'material-ui/svg-icons/navigation-arrow-drop-right'
 import { withRouter } from 'react-router-dom'
-
+import { connect } from 'react-redux'
 
 class DrawerOpenRightExample extends React.Component {
```
Comment (lehaSVV2009) [inline web/src/components/Home/DrawerOpenRightExample.jsx]: strange name.. Consider `LeftBar`

File: web/src/components/Home/DrawerOpenRightExample.jsx (hunk 1 above)
Comment (PavelEvleev) [reply web/src/components/Home/DrawerOpenRightExample.jsx]: done)

File: web/src/components/Home/DrawerOpenRightExample.jsx (hunk 2 above)
Comment (PavelEvleev) [reply web/src/components/Home/DrawerOpenRightExample.jsx]: done)

**Pull request merged**